src/template.ts

```diff
@@ -297,7 +330,7 @@ switch (data.method) {
     };
     const chunks = chunkPayload(
       payload,
-      ['objectIDs', 'positions'],
+      ['objectIDs', 'objectData', 'positions'],
```

Comment:
Just checking if `objectData` added here and below was intentional?

Reply:
Intentional. It should be chunked in the same manner as `objectJDs` and `positions`